system/jlib/jmetrics.hpp

```diff
@@ -136,14 +136,25 @@ class jlib_decl GaugeMetric : public MetricVal
     GaugeMetric(const char *name, const char *description) :
         MetricVal{name, description, MetricType::METRICS_GAUGE}  { }
 
+    void add(int64_t delta)
+    {
+        value += delta;
+    }
+
     /*
      * Update the value as indicated
      */
-    void add(int64_t delta)
+    void inc(uint64_t delta)
```

Comment:
minor: What is the benefit of the new functions, rather than calling add(amount) and add(-amount)?

Reply:
Jake asked for more meaningful method names.

Reply:
From your initial comment:

> Also changed the methods on the gauge to inc and dec. Left add temporarily for when the metric unit tests are merged. Once that is done I will remove it and update the test cases.

I don't think Jake made any comment on these functions.

Reply:
Most likely lost when rebased. Please get with him.

Reply:
As far as I can see this hasn't been reopened, so it was part of your initial commit.

Reply:
Jake says he did comment on it.  I would suggest one of the following

* have a single function, but rename it so it is neutral.  E.g. adjust()

* have two functions.  But parameters should be signed value so you can still say add(delta) where delta can be negative.

(I don't mind which.)

If you opt for 2 functions the original 3rd function should be deleted, because otherwise the public interface is confusing.

Reply:
not sure where the comment was, I thought in PR, but perhaps it was offline.
I think from offline chat a single method as it was, but with a more generic name, e.g ."adjust" is preferred.